python/utilities/dnslookup/dnslookup.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+# Implement DNS inclusion proof checking, see [TBD].
+#
+# Unfortunately, getting at the SCTs in general is hard in Python, so this
+# does not start with an SSL connection, but instead fetches a log entry by
+# index and then verifies the proof over DNS.
+
+import base64
+import dns.resolver
```

Comment:
Where does this module come from? It's not mentioned [here](https://docs.python.org/2.7/py-modindex.html)...